README.md

```diff
@@ -11,7 +11,7 @@ If you haven't used [Grunt](http://gruntjs.com/) before, be sure to check out th
 npm install grunt-html2js --save-dev
 ```
 
-One the plugin has been installed, it may be enabled inside your Gruntfile with this line of JavaScript:
```

Comment:
Hey, I'm attempting to open a PR to this project for issue #83 but getting a 403 error. Would you know if I need any additional permissions?

@karlgoldstein?